acme/jax/experiments/make_distributed_experiment.py

```diff
@@ -204,8 +208,11 @@ def build_learner(
   def build_inference_server(
       inference_server_config: inference_server_lib.InferenceServerConfig,
       variable_source: core.VariableSource,
+      server_number=0, # to distribute GPUs
```

Comment:
This makes it so we can't do multiple GPUs on the same inference server, which doesn't seem like what we want, considering there is pmap parallelism built in to inferenceserver. I think make this optional, if its None then we use all devices, and have that be the default.

Reply:
Maybe make another CLArg that says "distribute_gpus", and if its true we do the server-number thing. Sure.